.ci/manage-aur.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# No point in running this script if we don't have keys available
+set +x
+if [[ -z ${AUR_KEY+x} ]]; then
+    echo "No AUR SSH key available, backing off!"
+    exit 1
+else
+    mkdir /root/.ssh
+    chmod 700 /root/.ssh
+    echo "$AUR_KEY" >/root/.ssh/id_rsa
+    chmod 600 /root/.ssh/id_rsa
+fi
+set -x
+
+declare -a PACKAGES
+read -a PACKAGES <<<"$@" # fixme? mapfile -t / readarray -d both produced unusable results
+
+# shellcheck source=/dev/null
+source .ci/util.shlib
+export GIT_SSH_COMMAND="ssh -o StrictHostKeyChecking=accept-new"
+
+[[ -v "TMPDIR" ]] || TMPDIR="/tmp"
+mkdir -p "$TMPDIR/aur-push"
+
+if [ -v "PACKAGES[0]" ] && [ "${PACKAGES[0]}" == "all" ]; then
+    echo "AUR push of all managed packages requested."
+    UTIL_GET_PACKAGES PACKAGES
+fi
+
+# Check if the array of packages is empty
+if [ ${#PACKAGES[@]} -eq 0 ]; then
+    echo "No packages to push."
+    exit 0
+fi
+
+for package in "${PACKAGES[@]}"; do
+    unset VARIABLES
+    declare -A VARIABLES
+    if UTIL_READ_MANAGED_PACAKGE "$package" VARIABLES; then
+        if [[ -v CI_MANAGE_AUR ]] && [[ "$CI_MANAGE_AUR" != 1 ]]; then
```

Comment:
This always returns false

```suggestion
        if [[ -v "VARIABLES[CI_MANAGE_AUR ]" ]] && [[ "${VARIABLES[CI_MANAGE_AUR ]}" != "true" ]]; then
```